content/en/tracing/trace_collection/custom_instrumentation/python/ddtrace-api.md

```diff
@@ -0,0 +1,256 @@
+---
+title: Python Custom Instrumentation using ddtrace-api-py (pre-release)
+description: '(pre-release) Manually instrument your Python application to send custom traces to Datadog.'
+code_lang: dd-api-alpha
+type: multi-code-lang
+code_lang_weight: 1
+further_reading:
+    - link: 'tracing/other_telemetry/connect_logs_and_traces'
+      tag: 'Documentation'
+      text: 'Connect your Logs and Traces together'
+    - link: 'tracing/glossary/'
+      tag: 'Documentation'
+      text: 'Explore your services, resources, and traces'
+---
+
+*Note: the ddtrace-api Python package is in a pre-release state and may contain significant issues. If you'd rather use something more stable, use the [ddtrace API][9].*
```

Comment:
Worth wrapping this in a warning div, like DORA metrics: https://github.com/DataDog/documentation/blob/cb5309cc7a44fbe29207426501d66db9f04f5a8b/content/en/dora_metrics/_index.md?plain=1#L32, so this warning stands out more.

Reply:
```suggestion
<div class="alert alert-warning">*Note: the ddtrace-api Python package is in a pre-release state and may contain significant issues. If you'd rather use something more stable, use the [ddtrace API][9].*</div> 

````

Reply:
Also worth writing out some examples of issues to give an idea of what they could be.

Reply:
I reworded this to be less scary